pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -488,16 +488,37 @@ protected void setupDB(@NonNull Config config) {
 
         int maxConnections = config.getInt("portal.maxConnections");
         String dbUrl = config.getString("portal.dbUrl");
+        String dssDBUrl = config.getString("dss.dbUrl");
```

Comment:
I would extract `DSS` and `DSM` db connection openings into different methods with meaningful names, in better case I think each of it deserves its own class.